artiq/frontend/aqctl_corelog.py

```diff
@@ -78,6 +81,26 @@ async def get_logs(host):
                     level = logging.ERROR
                 name = 'firmware.' + m.group(2).replace('::', '.')
                 text = m.group(3)
+
+                # Add Channel name if any
+                n = re.match(r".+?(channel\s(\d+))", text)
```

Comment:
For async errors you probably want to extend the comm_kernel protocol and upload a map of friendly channel names to the device, for it to use later in logging.